pages/dkp/kommander/1.3/release-notes/1.3.2/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.3.2
+title: Release Notes Kommander 1.3.2
+menuWeight: 0
+excerpt: View release-specific information for Kommander 1.3.2
+enterprise: false
+---
+
+**D2iQ&reg; Kommander&reg; version 1.3.2 was released on 8, April 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Supported Versions
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
+
+# New features
+
+- Enabled PodSecurityPolicy in Kubecost.
+- Updated license page copy in Kommander UI.
+
+# Bug fixes
+
+- Updated UI to handle new KUDO param types.
```

Comment:
Do either of these have COPS numbers? Please add.

Reply:
No COPS for these